### PR TITLE
Release/2 0 0

### PR DESCRIPTION
**Pull request by jfriel-oqc**

### CODE REVIEW

**Review by hamidelmaazouz — APPROVED** [review]

File: src/tests/files/qasm/qasm2/long.qasm
Comment (keriksson-rosenqvist) [inline src/tests/files/qasm/qasm2/long.qasm:1]: This is exactly the same as [`long_qasm.qasm`](https://github.com/oqc-community/qat/blob/release/2_0_0/src/tests/files/qasm/qasm2/long_qasm.qasm). Is there a need for the duplication?

File: src/tests/serialised_compiler_config_templates/serialised_default_compiler_config_legacy.json
Comment (keriksson-rosenqvist) [inline src/tests/serialised_compiler_config_templates/serialised_default_compiler_config_legacy.json:1]: Duplicate of the `v02` file. We should only have one.

File: src/tests/serialised_compiler_config_templates/serialised_full_compiler_config_legacy.json
Comment (keriksson-rosenqvist) [inline src/tests/serialised_compiler_config_templates/serialised_full_compiler_config_legacy.json:1]: Duplicate of the `v02` file. We should only have one.

**Review by keriksson-rosenqvist — CHANGES_REQUESTED** [review]
Only minor comments but blocking so we ensure we make a release of current main before this gets merged.

**Comment by jfriel-oqc** [comment]
no longer relevant, closing

**Pull request closed without merging**